src/Education.UnitTests/SimpleRuleHelpXamlBuilderTests.cs

```diff
@@ -131,7 +131,7 @@ private static bool ProcessResource(string fullResourceName)
             try
             {
                 var data = ReadResource(fullResourceName);
-                var jsonRuleInfo = JsonConvert.DeserializeObject<RuleInfo>(data);
+                var jsonRuleInfo = LocalRuleMetadataProvider.RuleInfoJsonDeserializer.Deserialize(data);
```

Comment:
Some of the tests were deserializing the embedded json as part of the test, so they need to use the new `RuleInfo` deserializer.